apps/marketing/content/blog/early-adopters.mdx

```diff
@@ -0,0 +1,65 @@
+  <MdxNextImage
+    src="/blog/early.png"
+    width="1260"
+    height="630"
+    alt="XKCD: Bug"
+  />
+
+  <figcaption className="text-center">
+    "Being early is, uh, good." -Unknown
+  </figcaption>
+</figure>
+
+## Community-Driven Development
+
+As we ramp up hiring and development speed for Documenso, I want to discuss how we plan to build its core version.
+As we want to be a highly community-driven product, we want to capture the thoughts and ideas of said community in two ways:
+
+1. Everything we plan (i.e. documen.so/roadmap) and build (i.e. documen.so/repo) is public and open to comments and suggestions from everyone by design. We plan to have discussions, streams, and articles showcasing what we're up to, encouraging feedback and iterations on our plans.
+2. We're looking for 100 die-hard early adopters to get more deep hands-on feedback. If you want to be among the first to use and help shape Documenso, we have a special offer for you:
+
+## The Early Adopter Plan
+
+- All first 100 signups are entitled to the early adopter plan.
+- The plan includes everything we build in the next 12 months and unlimited<sup>1</sup> signatures
+- This plan is priced at $30/mo.
+- No matter what we add, the price is guaranteed for life as a thank you for supporting Documenso's mission<sup>2</sup>. And we plan to add a lot.
+- This also includes unlimited users<sup>3</sup> as part of the upcoming support for teams.
+- If you already claimed an early adopter account in the past and canceled, we are happy to reactivate your account with the early adopter pricing. Reach out to support@documenso.com
+
+## Being an Early Adopter
+
+Being eligible for the early adopter plan has no formal requirements like giving feedback or being active in the community. In good faith, we assume you sign the Documenso Supporter Pledge to be part of this. And if you want to use the newest version quietly, that's fine, too. Everything the die-hard community brings up will be carefully considered and prioritized <sup>4</sup>. Also, there will obviously be limited edition merch available for the first 100.
+Documenso currently runs the community reviewed 0.9.1 version. Getting from here to the globally loved and adopted signing tool we all deserve will take a lot of work, and we want you on board to help us create it. Join us in shaping the future of open signing and having fun doing it. [Malfunction Mania](https://documenso.com/blog/malfunction-mania) and releasing 1.0 will go a long way.
+
+## Extending our open metrics
+
+As part of our ongoing effort to be open and transparent in our doing, we are adding "Early Adopters" to our [/open page](https://documenso.com/open) page. After we exceed the early adopter slots, this metric will transition to "Customers". When no more early adopter seats can be claimed, the early adopter plan will transition to a standard paid plan. It will still be priced at $30/mo., but will no longer include upcoming features or unlimited seats.
```

Comment:
Need a footnote on this that it will receive some upcoming features but not those that would incur additional costs such as teams etc.

Reply:
- That makes sense. I deliberately made it sound a bit like it would get less so early adopters seem more enticing
- Will add